test/unit/range/views/view_minimiser_hash_test.cpp

```diff
@@ -40,8 +44,8 @@ struct iterator_fixture<iterator_type> : public ::testing::Test
     seqan3::dna4_vector text{"ACGGCGACGTTTAG"_dna4};
     result_t expected_range{26, 97, 27};
 
-    decltype(seqan3::views::minimiser_hash(text, seqan3::ungapped{4}, window_size{8}, seed{0})) test_range =
-         seqan3::views::minimiser_hash(text, seqan3::ungapped{4}, window_size{8}, seed{0});
+    using test_range_t = decltype(text | ungapped_view);
+    test_range_t test_range = text | ungapped_view;
```

Comment:
I was confused about this change, but I realize that ungapped_view is defined above as the same expression. 
Well spotted!